docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 7
+title: Auth control token API reference information
```

Comment:
```suggestion
title: Access control token API reference information
```

Reply:
I thought you wanted things to say Auth.